source/services/encryption/EncryptionService.stories.tsx

```diff
@@ -110,9 +117,28 @@ const testSymmetricKeySetup = async () => {
   printPublicKeyResult(coApplicantStina, updatedForm);
   await printSymmetricKeyResult(coApplicantStina, updatedForm);
 
-  await setupSymmetricKey(mainApplicantYlva, updatedForm);
+  updatedForm = await setupSymmetricKey(mainApplicantYlva, updatedForm);
   await printSymmetricKeyResult(mainApplicantYlva, updatedForm);
 
+  updatedForm = await encryptFormAnswers(mainApplicantYlva, updatedForm);
+  const encryptedFormCopy = JSON.parse(JSON.stringify(updatedForm));
```

Comment:
Maybe it would be nice if deep copy of object could be extraced to it's own function. Then this line would be easier to understand?

`specialSuperDeepAwesomeCopy(thing) { ... }`

Reply:
Moved into `Objects.tsx` as `deepCopyViaJson`.